javascript/src/handleEnrichmentWebhook.js

```diff
@@ -26,28 +26,11 @@ module.exports = asyncHandler(async function handleEnrichmentWebhook(req, res) {
     `Received Enrichment webhook - https://app.transcend.io${req.body.extras.request.link}`,
   );
 
-  // Add new identifers
-  const signedRequestIdentifiers = {
-    email: [
-      createEnricherJwt({
-        value: 'test+enriched@transcend.io',
-      }),
-      createEnricherJwt({
-        value: 'test+access@transcend.io',
-      }),
-    ],
-    phone: [
-      createEnricherJwt({
-        countryCode: 'US',
-        number: '+18609066012',
-      }),
-    ],
-  };
-
   // Check if we should place a hold on this request
   const requestIdentifier = signedBody.value;
```

Comment:
I think it'd be best to show the `req.body` path here

Reply:
I'll make this change as it will open up an opportunity to strip out some code in verify**AndExtract**Webhook